pkg/meta/base.go

```diff
@@ -203,7 +203,7 @@ func (m *baseMeta) refreshSession() {
 }
 
 func (m *baseMeta) CleanStaleSessions() {
-	sids, err := m.en.doFindStaleSessions(time.Now().Add(time.Minute * -5).Unix())
+	sids, err := m.en.doFindStaleSessions(time.Now().Add(time.Minute*-5).Unix(), 100)
```

Comment:
Right now, we can only do it once in a minute, we should increase the limit, for example 1000.

Reply:
OK